tensorflow/compiler/mlir/tensorflow/transforms/unroll_batch_matmul.cc

```diff
@@ -117,28 +117,15 @@ std::vector<Value> ConvertTFBatchMatMulOp<BatchMatMulOpType>::sliceInput(
     auto reshape_op = createReshapeOp(value, {batch_size, num_rows, num_cols},
                                       element_type, loc, rewriter);
 
-    // Create a constant op for the split axis (=0)
-    auto split_dimension_type =
-        RankedTensorType::get({}, rewriter.getIntegerType(32));
-    auto split_dimension_attr = DenseElementsAttr::get(split_dimension_type, 0);
-    auto split_dimension_op = rewriter.create<TF::ConstOp>(
-        loc, split_dimension_type, split_dimension_attr);
-
     // Split along each batch.
-    SmallVector<int64_t, 3> slice_size = {1, num_rows, num_cols};
+    SmallVector<int64_t, 3> slice_size = {num_rows, num_cols};
     Type slice_result_type = RankedTensorType::get(slice_size, element_type);
     llvm::SmallVector<Type, 4> output_types(batch_size, slice_result_type);
-    auto split_op = rewriter.create<TF::SplitOp>(loc, output_types,
-                                                 split_dimension_op.getOutput(),
-                                                 reshape_op.getOutput());
+    auto split_op = rewriter.create<TF::UnpackOp>(
```

Comment:
unpack_op

Reply:
Done in 843d9d64baa9bab72e8adb0de5b8644ff90ca53a